game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -222,6 +285,12 @@ function onStop() {
     sunsFan.RemoveNoDraw()
 
     clearNpcs()
+
+    if (pathParticleID) {
+        ParticleManager.DestroyParticle(pathParticleID, false)
+        ParticleManager.ReleaseParticleIndex(pathParticleID)
+        pathParticleID = undefined
+    }
```

Comment:
Should also clean up when the section ends (which is never supposed to happen, but we might change that (in the script changes PR I made a cleanup() function where that could be put which gets called both on skip and normal ending)

Reply:
Done.